r5/fhirpath/tests-fhir-r5.xml

```diff
@@ -459,6 +459,7 @@ Any text enclosed within is ignored
 	<group name="testSubSetOf">
 		<test name="testSubSetOf1" inputfile="patient-example.xml"><expression>Patient.name.first().subsetOf($this.name)</expression><output type="boolean">true</output></test>
 		<test name="testSubSetOf2" inputfile="patient-example.xml"><expression>Patient.name.subsetOf($this.name.first()).not()</expression><output type="boolean">true</output></test>
+		<test name="testSubSetOf3" inputfile="explanationofbenefit-example.json"><expression>supportingInfo.where(category.coding.code = 'additionalbodysite').sequence.subsetOf(item.informationSequence)</expression><output type="boolean">true</output></test>

```

Comment:
This test fails because the expression itself is wrong - it's missing a $this. in front of 'item'